test/unit/jest.config.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
 		[ `@wordpress\\/(${ transpiledPackageNames.join(
 			'|'
 		) })$` ]: 'packages/$1/src',
+		// See https://github.com/facebook/jest/issues/11100#issuecomment-967161978
+		'@eslint/eslintrc': '@eslint/eslintrc/dist/eslintrc-universal.cjs',
```

Comment:
Should we include this workaround in the Jest preset that is shared also with `@wordpress/scripts`:

https://github.com/WordPress/gutenberg/blob/trunk/packages/jest-preset-default/jest-preset.js

Reply:
I've put this workaround in the jest-preset-default package since all consumers of the preset will now benefit from it.
The unit tests still run (on my machine), hopefully CI agrees.

The issue should happen whenever someone tries to import eslint (v8) inside of Jest test files (e.g. when using `ruleTester`).

Reply:
It’s rather an edge case then, but it’s something that Jest 27 resolves so it’s fine to keep the workaround for now 👍